frontend/server/config.default.php

```diff
@@ -91,6 +91,14 @@
 try_define('OMEGAUP_ENABLE_REJUDGE_ON_PROBLEM_UPDATE', true);
 try_define('OMEGAUP_GRADER_FAKE', false);
 
+# ####################################
+# COURSE CLONE CONFIG
+# ####################################
+try_define(
+    'OMEGAUP_COURSE_CLONE_SECRET_KEY',
+    'om6f8xSU_xkrelmCTSahbbxl3PRovgAfkrThyrqQ9JesEegaup'
```

Comment:
```suggestion
    '6f8xSU_xkrelmCTSahbbxl3PRovgAfkrThyrqQ9JesE'
```

huh, _parece_ que no se pegó la llave correctamente: empieza con `om` y termina con `egaup`.

Reply:
Ups, ya lo corregí. 

Pero aún así sigue sin funcionar.  Quise generar una nueva secret key, pero ya no me funcionó el script que me compartiste. Recuerdo que hace rato si funcionaba. Ahora me manda el siguiente error:
```
ubuntu@6f8a27ffc7c2:/opt/omegaup/frontend/server/cmd$ php GeneratePasetoSymmetricKey.php
PHP Warning:  require_once(frontend/server/libs/third_party/sodium_compat/autoload.php): failed to open stream: No such file or directory in /opt/omegaup/frontend/server/cmd/GeneratePasetoSymmetricKey.php on line 5
PHP Fatal error:  require_once(): Failed opening required 'frontend/server/libs/third_party/sodium_compat/autoload.php' (include_path='.:/usr/share/php') in /opt/omegaup/frontend/server/cmd/GeneratePasetoSymmetricKey.php on line 5
```

Estuve revisando donde me manda el error y espera una llave con una longitud de 32 caracteres y lo que se está generando es de 31.